numpy/core/src/umath/absolute.dispatch.cpp

```diff
@@ -0,0 +1,92 @@
+/*@targets
+ * $maxopt $keep_baseline avx512_skx asimd
+ */
+
+#define _UMATHMODULE
+#define _MULTIARRAYMODULE
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+
+#undef HWY_TARGET_INCLUDE
+#define HWY_TARGET_INCLUDE "absolute.dispatch.cpp"  // this file
+#include <hwy/foreach_target.h>  // must come before highway.h
```

Comment:
This and the HWY_TARGET_INCLUDE can/should be commented out for static dispatch. We only want to compile the file once for one target.

Reply:
Eek, I guess that's just a result size issue? I'll clean it up though.